Realm/RLMRealm.h

```diff
@@ -423,11 +423,11 @@ typedef void (^RLMNotificationBlock)(RLMNotification notification, RLMRealm *rea
 
  @warning This method may only be called during a write transaction.
 
- @param array  An `RLMArray`, `NSArray`, or `RLMResults` of `RLMObject`s (or subclasses) to be deleted.
- 
+ @param array  An enumerable object such as `NSArray` or `RLMResults` which contains objects to be deleted.
```

Comment:
The doc-comments were changed to be consistent in phrasing between methods. Let me know if you notice a regression.